src/Parser/FrostMage/Modules/Cooldowns/FrozenOrb.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+import SpellIcon from 'common/SpellIcon';
+import { formatPercentage, formatNumber } from 'common/format';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import Module from 'Parser/Core/Module';
+
+class FrozenOrb extends Module {
```

Comment:
The FrozenOrb Module was something I was playing with to try and do cooldown reduction. I built some of it and never finished it because i figured i would wait until Zero has his cooldown tracking stuff implemented, and then i forgot to remove this file from the PR. Essentially though, the Statistic Box was Blizzard because i was testing the BlizzardHits portion to see if it was working. And Blizzard and Brain Freeze are in there because Blizzard Hits reduce the cooldown on FrozenOrb by 0.5 seconds and with T204pc, Brain Freeze Procs reduce the cooldown on Frozen Orb by 5 seconds. Either way ill remove this file